File: 1768-merge-strings-alternately/1768-merge-strings-alternately.go
func mergeAlternately(word1 string, word2 string) string {
    ans := []byte{}
    i, j := 0, 0
    for i < len(word1) || j < len(word2) {
        if i < len(word1) {
            ans = append(ans, word1[i])
            i += 1
        }

        if j < len(word2) {
            ans = append(ans, word2[j])
            j += 1
        }
    }    

    return string(ans)
}

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        """
        :type word1: str
        :type word2: str
        :rtype: str
        """
        i, j = 0, 0
        res = ""

        while i < len(word1) or j < len(word2):
            if i < len(word1):
                res += word1[i]
                i += 1

            if j < len(word2):
                res += word2[j]
                j += 1
        
        return res
        